style.css

```diff
@@ -1,145 +1,78 @@
 @import url("https://fonts.googleapis.com/css2?family=Dancing+Script&family=Handlee&family=Roboto:wght@100&display=swap");
+@import "./Components/base.css";
+@import "./Components/utility.css";
 
```

Comment:
All the other styles in your `style.css` file could be extracted to their own CSS files and renamed according to BEM.

Reply:
ok, thx a lot!